api-report/odsp-driver.api.md

```diff
@@ -38,7 +38,7 @@ export function encodeOdspFluidDataStoreLocator(locator: OdspFluidDataStoreLocat
 export function getApiRoot(origin: string): string;
 
 // @public (undocumented)
-export function getHashedDocumentId(driveId: string, itemId: string): string;
+export function getHashedDocumentId(driveId: string, itemId: string): Promise<string>;
```

Comment:
Please take a look at usage in Bohemia. I believe they are using it and logging as well, such that their events have same structure and we can do queries across tables